apps/docusaurus/docs/reference/indexer.md

```diff
@@ -0,0 +1,389 @@
+| metadata                    | Object  | Use the [Hasura explorer](#indexer-api-reference) to see fields for `metadata` in this table.                                                        |
+| owner_address               | String  | This is an Aptos account address that owns the asset. Ex. "0x123abc456def7890abcdef1234567890abcdef1234"                                             |
+| owner_aptos_names           | Table   | References [owner_aptos_names](#current_aptos_names).                                                                                                |
+| owner_aptos_names_aggregate | Table   | References [owner_aptos_names](#current_aptos_names).                                                                                                |
+| storage_id                  | String  | Identifier for the storage used in the transaction. Ex. "0xa815a9a09105973084bfc31530e7c8f002846787c2f0521e1e34dc144ad83b89"                         |
+| storage_refund_amount       | bigint  | Amount refunded for storage after the transaction. Ex. 50                                                                                            |
+| token_standard              | String  | Aptos standard that the collection adheres to. Ex. "v1"                                                                                              |
+| transaction_timestamp       | String  | Timestamp when the transaction occurred. Ex. "2024-04-17T02:14:25.68771"                                                                             |
+| transaction_version         | bigint  | Blockchain version of the transaction. Ex. 2                                                                                                         |
+| type                        | String  | Type of the transaction, described by a Move entry function. Ex. "0x3::token::TokenStore"                                                            |
+
+### `current_fungible_asset_balances`
+
+_Has an aggregate view for summary data called `current_fungible_asset_balances_aggregate`_
+
+This tracks the asset balances of each account on-chain. It includes v1 token data.
+
+| Field                      | Type    | Description                                                                                                                       |
+| -------------------------- | ------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| amount                     | bigint  | The amount of the asset owned. Ex. 2000                                                                                           |
+| asset_type                 | String  | The type of the asset, described by a Move resource. Ex. "0x1::aptos_coin::AptosCoin"                                             |
```

Comment:
For v2 this is the address of the fungible asset metadata object. For v1 it's the fully qualified path of the move resource.